application-scopes.json

```diff
@@ -1,48 +1,49 @@
-  "7b3f05d5-f68c-4b8d-8c59-a2ecd12f24af": "Microsoft Graph - DeviceManagementApps.ReadWrite.All (Read and write Microsoft Intune apps)",
+  "44642bfe-8385-4adc-8fc6-fe3cb2c375c3": "Microsoft Graph - DeviceManagementManagedDevices.ReadWrite.All (Read and write Microsoft Intune devices)",
+  "c2d95988-7604-4ba1-aaed-38a5f82a51c7": "Microsoft Graph - OnPremDirectorySynchronization.ReadWrite.All (Read and write all on-premises directory synchronization information)",
   "204e0828-b5ca-4ad8-b9f3-f32a958e7cc4": "Microsoft Graph - User.ReadWrite.All (Read and write all users' full profiles)",
-  "0b5d694c-a244-4bde-86e6-eb5cd07730fe": "Microsoft Graph - Domain.ReadWrite.All (Read and write domains)",
+  "7256e131-3efb-4323-9854-cf41c6021770": "Microsoft Graph - Policy.ReadWrite.IdentityProtection (Read and write your organization’s identity protection policy.)",
+  "4d135e65-66b8-41a8-9f8b-081452c91774": "Microsoft Graph - Policy.ReadWrite.ConsentRequest (Read and write consent request policy)",
+  "84bccea3-f856-4a8a-967b-dbe0a3d53a64": "Microsoft Graph - AppRoleAssignment.ReadWrite.All (Manage app permission grants and app role assignments)",
   "ad902697-1014-4ef5-81ef-2b4301988e8c": "Microsoft Graph - Policy.ReadWrite.ConditionalAccess (Read and write your organization's conditional access policies)",
+  "e4aa47b9-9a69-4109-82ed-36ec70d85ff1": "Microsoft Graph - AccessReview.ReadWrite.All (Manage access reviews that you can access)",
+  "ae7a573d-81d7-432b-ad44-4ed5c9d89038": "Microsoft Graph - EntitlementManagement.ReadWrite.All (Read and write entitlement management resources)",
+  "014b43d0-6ed4-4fc6-84dc-4b6f7bae7d85": "Microsoft Graph - Policy.ReadWrite.CrossTenantAccess (Read and write your organization's cross tenant access policies)",
+  "059e5840-5353-4c68-b1da-666a033fc5e8": "Microsoft Graph - ThreatSubmissionPolicy.ReadWrite.All (Read and write all threat submission policies)",
   "edd3c878-b384-41fd-95ad-e7407dd775be": "Microsoft Graph - Policy.ReadWrite.Authorization (Read and write your organization's authorization policy)",
-  "46ca0847-7e6b-426e-9775-ea810a948356": "Microsoft Graph - Organization.ReadWrite.All (Read and write organization information)",
-  "662ed50a-ac44-4eef-ad86-62eed9be2a29": "Microsoft Graph - DeviceManagementServiceConfig.ReadWrite.All (Read and write Microsoft Intune configuration)",
-  "e60370c1-e451-437e-aa6e-d76df38e5f15": "Skype and Teams Tenant Admin API - user_impersonation (Access Microsoft Teams and Skype for Business data as the signed in user)",
+  "89fe6a52-be36-487e-b7d8-d061c450a026": "Microsoft Graph - Sites.ReadWrite.All (Edit or delete items in all site collections)",
   "4f5bc9c8-ea54-4772-973a-9ca119cb0409": "Microsoft Graph - Policy.ReadWrite.AccessReview (Read and write your organization's directory access review default policy)",
-  "ef4b5d93-3104-4664-9053-a5c49ab44218": "Microsoft Graph - Agreement.ReadWrite.All (Read and write all terms of use agreements)",
```

Comment:
Would need to re-run the script to generate this file after the scopes are re-added, I think